.circleci/config.yml

```diff
@@ -64,12 +64,24 @@ workflows:
           filters:
             tags:
               only: /.*/
+      - publish_shared:
+          requires:
+            - node6
+            - node8
+            - node9
+            - node10
+          filters:
+            branches:
+              ignore: /.*/
+            tags:
+              only: /^v[\d.]+$/
       - publish_npm:
           requires:
             - node6
```

Comment:
no need to have these `node*` dependencies if it depends on `publish_shared`. It just makes the graph look weird. Of if you want parallel publishing, then remove the `publish_shared` dependency.